dev/io.openliberty.org.apache.taglibs.standard.3.0/bnd.bnd

```diff
@@ -25,6 +25,9 @@ Import-Package: \
    !org.eclipse.tags.shaded.org.apache.xml.serialize,\
    !org.eclipse.tags.shaded.org.apache.xml.serializer,\
    !org.apache.xerces*, \
+   jakarta.servlet.jsp.jstl.*;version="[3.0,4.0)", \
+   jakarta.servlet.jsp.*;version="[3.1,4.0)", \
+   jakarta.servlet.*;version="[6.0,7.0)", \
```

Comment:
Sorry to keep dragging this PR along but I compared the 2.0 bundle and saw that the versions could also be updated there, too. Maybe we could create an issue instead and follow up later on since it's not directly related to EE10? 

Version ends should be knocked down slightly. 

https://github.com/OpenLiberty/open-liberty/blob/a8ca7e607a54727c5e477ab49315fcd3c4a17ba8/dev/io.openliberty.org.apache.taglibs.standard/bnd.bnd#L23-L25